[2018-03-08 14:16:09][INFO ][main][org.springframework.boot.StartupInfoLogger.logStarting:48]:Starting TestSpringBoot on USER-20160201BI with PID 5024 (E:\work\xcs\xcs-demo\target\classes started by Administrator in E:\work\xcs\xcs-demo)
[2018-03-08 14:16:09][ERROR][main][com.xchushi.fw.transfer.sender.HttpSender.<init>:73]:initHttpSender fail:initCollectSendExecutor fail:Assert fail, bl isn't true
com.xchushi.fw.common.exception.InitException: initCollectSendExecutor fail:Assert fail, bl isn't true
	at com.xchushi.fw.transfer.runner.DefalutCollectSendRunner.<init>(DefalutCollectSendRunner.java:69)
	at com.xchushi.fw.transfer.sender.HttpSender.initHttpSender(HttpSender.java:115)
	at com.xchushi.fw.transfer.sender.HttpSender.<init>(HttpSender.java:71)
	at com.xchushi.fw.transfer.sender.HttpSender.<init>(HttpSender.java:65)
	at com.xchushi.fw.transfer.sender.HttpSender.<init>(HttpSender.java:61)
	at com.xchushi.fw.transfer.sender.HttpSender.getSender(HttpSender.java:154)
	at com.xchushi.fw.transfer.sender.SenderFactory.getSender(SenderFactory.java:6)
	at com.xchushi.fw.log.elasticsearch.logger.TCPEsLogger.<init>(TCPEsLogger.java:39)
	at com.xchushi.fw.log.elasticsearch.logger.TCPEsLogger.getLogger(TCPEsLogger.java:53)
	at com.xchushi.fw.log.XcsLoggerFactory.getLogger(XcsLoggerFactory.java:10)
	at com.xchushi.fw.log.logback.XcsLogbackAppender.initXcsLogbackLogger(XcsLogbackAppender.java:54)
	at com.xchushi.fw.log.logback.XcsLogbackAppender.append(XcsLogbackAppender.java:31)
	at com.xchushi.fw.log.logback.XcsLogbackAppender.append(XcsLogbackAppender.java:17)
	at ch.qos.logback.core.AppenderBase.doAppend(AppenderBase.java:82)
	at ch.qos.logback.core.spi.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:51)
	at ch.qos.logback.classic.Logger.appendLoopOnAppenders(Logger.java:270)
	at ch.qos.logback.classic.Logger.callAppenders(Logger.java:257)
	at ch.qos.logback.classic.Logger.buildLoggingEventAndAppend(Logger.java:421)
	at ch.qos.logback.classic.Logger.filterAndLog_0_Or3Plus(Logger.java:383)
	at ch.qos.logback.classic.Logger.log(Logger.java:765)
	at org.apache.commons.logging.impl.SLF4JLocationAwareLog.info(SLF4JLocationAwareLog.java:155)
	at org.springframework.boot.StartupInfoLogger.logStarting(StartupInfoLogger.java:48)
	at org.springframework.boot.SpringApplication.logStartupInfo(SpringApplication.java:579)
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:341)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:301)
	at com.test.springboot.TestSpringBoot.main(TestSpringBoot.java:11)
Caused by: java.lang.IllegalArgumentException: Assert fail, bl isn't true
	at com.xchushi.fw.common.Asset.isTrue(Asset.java:27)
	at com.xchushi.fw.common.Asset.isTrue(Asset.java:32)
	at com.xchushi.fw.common.util.SimpleFileQueue.<init>(SimpleFileQueue.java:45)
	at com.xchushi.fw.common.util.SimpleFileQueue.<init>(SimpleFileQueue.java:33)
	at com.xchushi.fw.transfer.runner.DefalutCollectSendRunner.initCollectSendExecutor(DefalutCollectSendRunner.java:82)
	at com.xchushi.fw.transfer.runner.DefalutCollectSendRunner.<init>(DefalutCollectSendRunner.java:67)
	... 25 common frames omitted
[2018-03-08 14:16:09][INFO ][main][org.springframework.boot.SpringApplication.logStartupProfileInfo:597]:The following profiles are active: default
[2018-03-08 14:16:10][INFO ][main][org.springframework.context.support.AbstractApplicationContext.prepareRefresh:581]:Refreshing org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@74c4ff15: startup date [Thu Mar 08 14:16:10 CST 2018]; root of context hierarchy
[2018-03-08 14:16:10][INFO ][background-preinit][org.hibernate.validator.internal.util.Version.<clinit>:30]:HV000001: Hibernate Validator 5.3.5.Final
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][ERROR][pool-1-thread-1][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][ERROR][pool-1-thread-2][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:10][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][ERROR][pool-1-thread-4][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:10][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:10][ERROR][pool-1-thread-3][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][main][org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainer.initialize:89]:Tomcat initialized with port(s): 8077 (http)
[2018-03-08 14:16:11][INFO ][main][org.apache.juli.logging.DirectJDKLog.log:179]:Starting service [Tomcat]
[2018-03-08 14:16:11][INFO ][main][org.apache.juli.logging.DirectJDKLog.log:179]:Starting Servlet Engine: Apache Tomcat/8.5.16
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-5][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-7][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-7][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-6][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-6][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.apache.juli.logging.DirectJDKLog.log:179]:Initializing Spring embedded WebApplicationContext
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.prepareEmbeddedWebApplicationContext:276]:Root WebApplicationContext: initialization completed in 1624 ms
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-1][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-1][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-8][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-8][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.web.servlet.ServletRegistrationBean.onStartup:190]:Mapping servlet: 'dispatcherServlet' to [/]
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.web.servlet.AbstractFilterRegistrationBean.configure:258]:Mapping filter: 'characterEncodingFilter' to: [/*]
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.web.servlet.AbstractFilterRegistrationBean.configure:258]:Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.web.servlet.AbstractFilterRegistrationBean.configure:258]:Mapping filter: 'httpPutFormContentFilter' to: [/*]
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-2][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-2][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][localhost-startStop-1][org.springframework.boot.web.servlet.AbstractFilterRegistrationBean.configure:258]:Mapping filter: 'requestContextFilter' to: [/*]
[2018-03-08 14:16:11][INFO ][pool-1-thread-4][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-5][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-3][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-3][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][ERROR][pool-1-thread-4][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:97]:I/O exception (java.net.NoRouteToHostException) caught when processing request to {}->http://120.76.98.17:7777: No route to host: connect
[2018-03-08 14:16:11][INFO ][pool-1-thread-5][org.apache.http.impl.execchain.RetryExec.execute:113]:Retrying request to {}->http://120.76.98.17:7777
[2018-03-08 14:16:11][ERROR][pool-1-thread-5][com.xchushi.fw.transfer.sender.HttpSender.sendingFailed:201]:No route to host: connect
java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:61)
	at com.xchushi.fw.common.util.HttpClientUtils.sendRequest(HttpClientUtils.java:44)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:232)
	at com.xchushi.fw.transfer.sender.HttpSender.synSend(HttpSender.java:211)
	at com.xchushi.fw.transfer.sender.HttpSender$1.call(HttpSender.java:175)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
